linkml_model/model/schema/meta.yaml

```diff
@@ -512,6 +512,100 @@ slots:
     in_subset:
       - basic
 
+  enum_uri:
+    aliases:
+      - public ID
+    domain: enum_definition
+    range: uriorcurie
+    description: URI of the enum in an RDF environment
+    ifabsent: class_curie
+    in_subset:
+      - basic
+      
+  include:
+    description: An enum expression that yields a list of permissible values that are to be included, after subtracting the minus set
+    domain: enum_expression
+    range: anonymous_enum_expression
+    multivalued: true
+
+  minus:
+    description: An enum expression that yields a list of permissible values that are to be subtracted from the enum
+    domain: enum_expression
+    range: anonymous_enum_expression
+    multivalued: true
+
+  inherits:
+    description: An enum definition that is used as the basis to create a new enum
+    comments:
+      - All permissible values for all inherited enums are copied to form the initial seed set
+    domain: enum_expression
+    range: enum_definition
+    multivalued: true
+
+  matches:
+    description: Specifies a match query that is used to calculate the list of permissible values
+    domain: enum_expression
+    range: match_query
+
+  identifier_pattern:
+    description: A regular expression that is used to obtain a set of identifiers from a source_ontology to construct a set of permissible values
+    domain: match_query
+    range: string
+    
+  concepts:
+    description: A list of identifiers that are used to construct a set of permissible values
```

Comment:
I'm a little confused. It says "a list of identifiers". Isn't that the same thing as "a set of permissible values"? This seems like something that should be statically defined, so I am confused by this saying "used to construct...". If the list of identifiers are extensionally provided here, then what is there left to construct?